include/net/fota_download.h

```diff
@@ -97,7 +97,19 @@ int fota_download_init(fota_download_callback_t client_callback);
  *
  * @param host Name of host to start downloading from. Can include scheme
  *             and port number, e.g. https://google.com:443
- * @param file Filepath to the file you wish to download.
+ * @param file Filepath to the file you wish to download. May be either a
+ *              single file path, relative to HOSTNAME (e.g. "path/to/binary.bin"),
+ *              or, if bootloader FOTA is enabled, can be two paths (both relative
+ *              to HOSTNAME), separated by a space (e.g. "path/to/s0.bin path/to/s1.bin").
+ *              \n \n 
```

Comment:
What's the `\n \n` for?

Reply:
They were intended to create better readability by separating key paragraphs, but either Breathe or Sphinx doesn't respond well to them, so I have removed them. The documentation is now a single paragraph.